lib/navigator/common_navigator.dart

```diff
@@ -40,19 +39,16 @@ class CommonNavigator {
     builder: (ctx, state) => ChangePasswordConfirmationPage(),
   );
 
-  static GoRoute login = GoRoute(
-    name: "login",
+  static GoRoute sign = GoRoute(
+    name: "sign",
     path: "/sign",
-    redirect: (context, state) => Guard.guards(context, [Guard.checkUnauthenticated]),
-    builder: (ctx, state) => LoginPage(),
-    routes: [lostPassword, register],
-  );
-
-  static GoRoute register = GoRoute(
```

Comment:
You should keep the register link but only to redirect to the sign page since it is used in the doc website